agent/api/container/container.go

```diff
@@ -61,9 +61,17 @@ const (
 	// v4 metadata endpoint
 	MetadataURIEnvVarNameV4 = "ECS_CONTAINER_METADATA_URI_V4"
 
+	// MetadataURIEnvVarNameGQL defines the name of the environment
+	// variable in containers' config, which can be used by the containers to access the
+	// GQL metadata endpoint
+	MetadataURIEnvVarNameGQL = "GQL_METADATA_URI"
```

Comment:
can we change the value to `ECS_CONTAINER_METADATA_URI_GQL` to make it a bit more consistent with the others?